index.js

```diff
@@ -62,7 +62,7 @@ function morgan (format, options) {
 
   if (format && typeof format === 'object') {
     opts = format
-    fmt = opts.format || 'default'
+    fmt = opts.format || 'combined'
```

Comment:
So this will break existing logs due to the format change, right? The date formats are different between default and combined, so you can't just change the default value of format.